openflow15/group.go

```diff
@@ -20,6 +20,15 @@ const (
 	 */
 )
 
+const (
+	OFPG_BUCKET_MAX   = 0xffffff00 /* Last usable bucket ID. */
```

Comment:
```suggestion
	OFPG_BUCKET_MAX   = 0xffffff00 /* Max usable bucket ID. */
```

Reply:
How to use this constant, e.g., is it only used in OFPGC_INSERT_BUCKET and OFPGC_REMOVE_BUCKET commands?

Reply:
It should be set as the value of CommandBucketId in GroupMod when using  OFPGC_INSERT_BUCKET and OFPGC_REMOVE_BUCKET commands.

Reply:
This comment is from the openflow file, I think we should reserve the original comment.

/* Bucket Id can be any value between 0 and OFPG_BUCKET_MAX */
enum ofp_group_bucket {
    OFPG_BUCKET_MAX   = 0xffffff00, /* Last usable bucket ID. */
    OFPG_BUCKET_FIRST = 0xfffffffd, /* First bucket ID in the list of action
                                       buckets of a group. This is applicable
                                       for OFPGC_INSERT_BUCKET and
                                       OFPGC_REMOVE_BUCKET commands. */
    OFPG_BUCKET_LAST  = 0xfffffffe, /* Last bucket ID in the list of action
                                          buckets of a group. This is applicable
                                          for OFPGC_INSERT_BUCKET and
                                       OFPGC_REMOVE_BUCKET commands. */
   OFPG_BUCKET_ALL = 0xffffffff  /* All action buckets in a group,
                                     This is applicable for
                                     only OFPGC_REMOVE_BUCKET command. */
};